### PR TITLE
fix some grammar 

### PR DESCRIPTION
**Pull request by richarda23**
# pull request

minor grammatical and English punctuation corrections 

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

**Review by ericbalawejder — COMMENTED** [review]
Looks good. Line 8 of `concepts/switch-statement/introduction.md` has an extra white space where the word "one" was removed.
```
case : you will use this  to declare
```

File: concepts/switch-statement/introduction.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
 
-Some keywords are usefull when using a switch statement.
+Some keywords are useful when using a switch statement.
 
-- `switch` : this keyword allow you to declare the structure of the switch. It his follow by the expression or the variable that will make the result change.
-- `case` : you will use this one to declare the differents possibilties for the result.
-- `break` : the `break` keyword is very usefull in order to stop the execution of the switch at the end of the wanted flow. If you forget it, the program will continue and may lead to unexpected results.
-- `default` : as it's name says use it as a default result when no other case matchs your expression's result.
+- `switch` : this keyword allows you to declare the structure of the switch. It is followed by the expression or the variable that will make the result change.
+- `case` : you will use this  to declare the differents possibilties for the result.
```
Comment (mirkoperillo) [inline concepts/switch-statement/introduction.md:8]:
```suggestion
- `case` :  you will use this to declare the differents possibilties for the result.
```

**Review by mirkoperillo — COMMENTED** [review]
@richarda23 I have suggested the modification proposed by @ericbalawejder  in comments

After this  little fix I think the PR is ready to be merged
Thank you for your work

**Review by ericbalawejder — APPROVED** [review]
Looks good. Thanks @mirkoperillo

**Comment by ericbalawejder** [comment]
Should we address the linter failure with another PR?

The `source_url` value is the empty string, but it must be a valid URL:
/home/runner/work/java/java/exercises/practice/robot-simulator/.meta/config.json

**Comment by mirkoperillo** [comment]
These lint errors are a nightmare :rofl: 
This is a very limited error so I think we can fix it in this PR to be more agile

**Pull request merged**